docs/src/main/paradox/docs/delta/api/resources-api.md

```diff
@@ -236,8 +240,38 @@ GET /v1/resources/{org_label}/{project_label}?from={from}
                                              &q={search}
                                              &sort={sort}
 ```
-                                          
-where...
+
+### Within an organization
```

Comment:
It is worth mentioning here that the results will contain only the resources from projects where the caller has access (`resources/read`)